src/main/java/br/com/cinemenu/cinemenuapi/rest/repository/PreviewMediaRepository.java

```diff
@@ -107,7 +112,46 @@ public PreviewMediaResponsePage getGenrePreviewMediaResponse(List<Integer> genre
         return new PreviewMediaResponsePage(page, mediaList, totalPages);
     }
 
+    public PreviewPopularResults getPeopleListResults(Integer page) {
+        verifyPage(page);
+
+        URI uri = URI.create(
+                TMDB_BASE_URL
+                        + TMBD_POPULAR_PERSON
+                        + apiKey
+                        + TMDB_LANGUAGE_PT_BR
+                        + TMDB_BASE_PAGE + page
+        );
+
+        return restTemplate.getForObject(uri, PreviewPopularResults.class);
+    }
+
+    public PreviewActorCreditsListResults getMovieListByActorId(Long id) {
+        verifyId(id);
+
+        URI uri = URI.create(
+                TMDB_BASE_URL
+                        + TMDB_ACTOR_MOVIE_LIST.formatted(id) + apiKey
+                        + TMDB_LANGUAGE_PT_BR
+        );
+
+        try {
+            return restTemplate.getForObject(uri, PreviewActorCreditsListResults.class);
+        } catch (HttpClientErrorException ex) {
+            throw new TMDBNotFoundException("id not found");
```

Comment:
acredito que adicionar o id na mensagem pode facilitar na inspeção caso a Exception aconteça
`id not found: <id>`